Vagrantfile

```diff
@@ -478,4 +478,41 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       puppet.environment = "main"
     end
   end
+
+  #
+  # Debian 8 boxes
+  #
+
+  # Box that installs ODL via its Ansible role on Debian 8
+  # This uses the default repo-based deb install method
+  config.vm.define "debian8_ansible" do |debian8_ansible|
+    # Build Vagrant box based on Debian 8
+    debian8_ansible.vm.box = "debian/jessie64"
+    debian8_ansible.vm.box_version = "=8.5.2"
```

Comment:
Maybe we should build it based on latest, so this doesn't need to be maintained to stay recent?